src/Util/Strings.php

```diff
@@ -0,0 +1,408 @@
+<?php
+/**
+ * @file src/Util/Strings.php
+ */
+
+namespace Friendica\Util;
+
+use Friendica\Content\ContactSelector;
+use Friendica\Core\Logger;
+
+/**
+ * @brief This class handles string functions
+ */
+class Strings
+{
+    /**
+     * @brief Generates a pseudo-random string of hexadecimal characters
+     *
+     * @param int $size
+     * @return string
+     */
+    public static function getRandomHex($size = 64)
+    {
+        $byte_size = ceil($size / 2);
+
+        $bytes = random_bytes($byte_size);
+
+        $return = substr(bin2hex($bytes), 0, $size);
+
+        return $return;
+    }
+
+    /**
+     * This is our primary input filter.
+     *
+     * The high bit hack only involved some old IE browser, forget which (IE5/Mac?)
+     * that had an XSS attack vector due to stripping the high-bit on an 8-bit character
+     * after cleansing, and angle chars with the high bit set could get through as markup.
+     *
+     * This is now disabled because it was interfering with some legitimate unicode sequences
+     * and hopefully there aren't a lot of those browsers left.
+     *
+     * Use this on any text input where angle chars are not valid or permitted
+     * They will be replaced with safer brackets. This may be filtered further
+     * if these are not allowed either.
+     *
+     * @param string $string Input string
+     * @return string Filtered string
+     */
+    public static function removeTags($string)
+    {
+        return str_replace(["<", ">"], ['[', ']'], $string);
+    }
+
+    /**
+     * @brief Use this on "body" or "content" input where angle chars shouldn't be removed,
+     * and allow them to be safely displayed.
+     * @param string $string
+     * 
+     * @return string
+     */
+    public static function escapeTags($string)
```

Comment:
Please rename to `escapeHtml` as this will escape all HTML special characters instead of just tags. I suggest you start renaming this method before renaming `removeTags`!